files/install

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-version=$(curl -sSL https://raw.githubusercontent.com/cisco-sso/kdk/master/cmd/root.go | grep -e 'kdk.Version' | grep "[0-9]" | sed 's|.*"\(.*\)"|\1|g')
+version=$(curl -sSL "https://api.github.com/repos/cisco-sso/kdk/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
The quotes around the URL aren't necessary. 

Should: remove them. (unless this is a windows compat thing)